VideoWeb/VideoWeb/ClientApp/src/app/shared/models/participant-media-status.ts

```diff
@@ -0,0 +1,4 @@
+export class ParticipantMediaStatus {
+    // tslint:disable-next-line: variable-name
+    constructor(public is_local_muted: boolean) {}
```

Comment:
should we add audio to the name?
`is_local_audio_muted` ?

Reply:
I believe so because we also will want to know when the camera is off, and that's classed as video muted